08-regression2.Rmd

```diff
@@ -342,17 +364,23 @@ so the "flat" extrapolation of K-NN likely does not match reality.
 ## Multivariate linear regression
 
 As in K-NN classification and K-NN regression, we can move beyond the simple
-case of one response variable and only one predictor and perform multivariate
-linear regression where we can have multiple predictors. In this case we fit a
-plane to the data, as opposed to a straight line.
-
+case of only one predictor to the case with multiple predictors,
+known as *multivariate linear regression*. 
 To do this, we follow a very similar approach to what we did for
-K-NN regression; but recall that we do not need to use cross-validation to choose any parameters,
+K-NN regression: we just add more predictors to the model formula in the 
+recipe. But recall that we do not need to use cross-validation to choose any parameters,
 nor do we need to standardize (i.e., center and scale) the data for linear regression. 
-We demonstrate how to do this below using the Sacramento real estate data with both house size
+Note once again that we have the same concerns regarding multiple predictors
+as in the settings of multivariate K-NN regression and classification: more predictors is **not** always
+better. But because the same predictor selection 
+algorithm from the classification chapter extends to the setting of linear regression,
+it will not be covered again in this chapter.
+
+We will demonstrate multivariate linear regression using the Sacramento real estate 
+data with both house size
 (measured in square feet) as well as number of bedrooms as our predictors, and
-continue to use house sale price as our outcome/target variable that we are
-trying to predict. We will start by changing the formula in the recipe to 
+continue to use house sale price as our response variable. We will start by 
+changing the formula in the recipe to 
 include both the `sqft` and `beds` variables as predictors:
 
 ```{r 08-lm-mult-test-train-split}
```

Comment:
we reuse the variable names for `lm_recipe` and `lm_fit` - we should avoid this practice so we don't confuse the readers... what about using `mlm_recipe` and `mlm_fit`?

Reply:
good idea, done